common/src/components/Footer/Footer.tsx

```diff
@@ -58,7 +59,6 @@ const Footer: React.FunctionComponent = () => {
       className="footer"
       data-h2-border="b(gray, top, solid, s)"
       data-h2-bg-color="b(lightgray[.6])"
-      style={{ marginTop: "auto" }}
```

Comment:
You need to add this back in 😅. It keeps the Footer at the bottom of the page. 
![image](https://user-images.githubusercontent.com/22059495/145482911-20ffd235-caa6-4a8b-a68e-a143ff35f251.png)